templates/duplicates.html.tt2

```diff
@@ -0,0 +1,243 @@
+    <script src="[% c.url_for("/js/duplicates.js?$version") %]" type="text/JAVASCRIPT"></script>
+    <style>
+        /* Table styling */
+        table {
+            width: 100%;
+            border-collapse: collapse;
+            font-family: Geist, sans-serif;
+            font-size: 10pt;
+            margin-top: 20px;
+            margin:auto;
+        }
+
+        /* Header styling */
+        table thead tr {
+            color: white;
+            font-weight: bold;
+        }
+
+        table thead th {
+            padding: 12px;
+            text-align: center;
+        }
+
+        /* Zebra striping */
+        table tbody tr:nth-child(2n+1) {
+            background-color: #363940;
+        }
+
+        /* Cell styling */
+        table tbody td {
+            padding: 10px;
+            text-align: center;
+            vertical-align: middle;
+        }
+
+        /* Thumbnail image */
+        table .thumbnail {
+            width: 50px;
+            height: auto;
+            border-radius: 4px;
+        }
+
+        /* Separator row */
+        .separator {
+            border-bottom: 2px solid #ddd;
+        }
+
+        /* Buttons */
+        .action-button {
+            margin: 0 5px;
+            padding: 5px 10px;
+            font-size: 9pt;
+            min-width: 50px;
+        }
+
+        .btn-danger {
+            background-color: #d9534f;
+            color: white;
+            border: none;
+        }
+
+        .btn-danger:hover {
+            background-color: #c9302c;
+        }
+
+        .thumbnail-wrapper {
+            margin: auto;
+            overflow: visible; /*hidden;*/
+            position: relative;
+            text-align: center;
+        }
+
+        /* Popover */
+        .thumbnail-popover {
+            display: none;
+            position: absolute;
+            top: -10px;
+            left: 60px;
+            width: 350px;
+            padding: 5px;
+            z-index: 100;
+        }
+
+        /* Popover image */
+        .thumbnail-popover img {
+            width: 100%;
+            height: auto;
+            border-radius: 4px;
+        }
+
+        .tag-column {
+            max-width: 700px;
+            text-align: center;
+            margin: auto;
+            max-height: 80px;
+            white-space: normal;
+            text-wrap: auto;
+            overflow: hidden;
+            text-overflow: ellipsis;
+        }
+
+        .table-control-wrapper {
+            display: flex;
+            justify-content: space-between;
+            padding: 15px;
+            font-family: Geist, sans-serif;
+            font-size: 13px;
+        }
+
+        .duplicate-control-btn-group {
+            margin-bottom: 15px;
+        }
+    </style>
+</head>
+
+<body>
+	<div class='ido' style='text-align:center; overflow-x:auto;'>
+		<h2 class="ih" style="text-align:center">Duplicates</h2>
+        <p>Found [% duplicates.size %] duplicate groups</p>
```

Comment:
There should be some explanatory text here that details how the feature works. 

("Found X duplicate groups" might be good to hide if there's no duplicate results? Not necessary though imo)